cobc/flag.def

```diff
@@ -232,6 +232,16 @@ CB_FLAG (cb_listing_symbols, 1, "tsymbols",
 CB_FLAG (cb_listing_cmd, 1, "tcmd",
 	_("  -ftcmd                specify command line in listing"))
 
-CB_FLAG_ON (cb_diagnostic_show_option, 1, "diagnostics-show-option",
+CB_FLAG_ON (cb_diagnostics_show_option, 1, "diagnostics-show-option",
 	_("  -fno-diagnostics-show-option\tsuppress output of option that directly\n"
 	  "                        controls the diagnostic"))
+
+CB_FLAG (cb_diagnostics_show_caret, 1, "diagnostics-show-caret",
```

Comment:
If possible please change to `CB_FLAG_ON` and add the reverse to the flags in tests/atlocal.in